docs/src/platform-support.md

```diff
@@ -0,0 +1,42 @@
+# Platform Support
+The `rust-gpu` project currently supports a limited number of platforms and graphics APIs. Right now we're not distributing build artifacts and we're primarily focused on the development of the project, so this is based on the current `main` branch. There are a lot of different configurations and hardware out there to support, this document is intended to document what is currently supported, what we intend to support, and what isn't supported. Over time as the project stabilises and our CI improves, more platforms and APIs will be supported and tested. Currently support for each topic is divided into the following three categories.
+
+- **Primary —** Built and tested on CI.
+- **Secondary —** Built but *not fully tested* on CI.
+- **Tertiary —** Present in the codebase but not built or tested.
+
+## Operating System
+
+| Operating System | Version | Support | Notes
+|-------------------|---------|---------|-------
+| Windows | 10+ | Primary | |
+| Linux | ??? | Primary | |
```

Comment:
maybe the ubuntu CI version we use here?

Reply:
That's a good idea and could be a base